mlibrary/search
# [LIBSEARCH-951] Radio buttons in advanced search screens for and/or/not are not selectable

# Overview
In Advanced search, when looking at all the datastores except for `Everything`, you are not able to click on the `AND`/`OR`/`NOT` options. However, you can navigate by keyboard.

`MultipleChoice` has been replaced by vanilla radio button options. A couple of WCAG errors and warnings have also been resolved.

This pull request fixes [LIBSEARCH-951](https://mlit.atlassian.net/browse/LIBSEARCH-951).

## Testing
- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
- Go to Advanced search.
  - Are you able to click on the radio buttons?
  - Do the radio button values get submitted when making a search?
  - Can you still navigate by keyboard?
